sale_stock_return_request/tests/test_stock_return_request.py

```diff
@@ -151,14 +151,12 @@ def test_01_return_sale_stock_from_customer(self):
         )
         # Test the view action
         res = self.return_request_customer.action_view_sales()
-        self.assertDictContainsSubset(
-            {
-                "type": "ir.actions.act_window",
-                "res_model": "sale.order",
-                "domain": "[('id', 'in', %s)]" % (sorted(sale_orders.ids)),
-            },
-            res,
-        )
+        expected_res = {
+            "type": "ir.actions.act_window",
+            "res_model": "sale.order",
+            "domain": "[('id', 'in', %s)]" % (sorted(sale_orders.ids)),
+        }
+        self.assertEqual(res, {**res, **expected_res})
```

Comment:
I don't get this assert.

Reply:
Yes, I had to check it as well, but it makes sense in the context of `assertDictContainsSubset()` being now obsolete (https://stackoverflow.com/a/59777678/3274287)

Reply:
At least a comment is needed to understand this line and the other that is the same.